CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_netsuite v0.19.0
+
+## Breaking Changes (full refresh required)
+- Added optional `fiscalcalendar` source table to support accurate fiscal year start dates (currently defaulted to calendar year). This table, related models (`stg_netsuite2__fiscal_calendar_tmp` and `stg_netsuite2__fiscal_calendar`), and relevant adjustments within `int_netsuite2__accounting_periods` are disabled by default. To enable this feature:
+  - Quickstart users: enable the fiscalcalendar table in the connection schema tab.
```

Comment:
I noticed in the Fivetran schema tab this table is capitalized as FiscalCalendar. Should we update with this casing syntax or do we think this is ok?

Reply:
Same comment from the source. Not necessary to change the casing since this is how the table will look in the destination. As long as it's the same casing in the quickstart.yml that's what matters most.